addons/coupon/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': "Coupon",
+    'summary': "Use discount coupons in different sales channels.",
+    'description': """Integrate coupon mechanism in orders.""",
+    'category': 'Sales',
+    'version': '1.0',
+    'depends': ['account', 'sales_team'],
```

Comment:
I am not sure those dependencies are required, can't we just put ['product']?

Reply:
I considered this before. I think I rejected the idea because of the taxes field in the product. But after checking, they appear to be set to false, so I guess 'product' as dependency is enough after all.

Reply:
lol i just saw this comment now after pushing my review suggesting to do just that. I honestly prefer having `account` as a dep because of the override i mentioned in the review, but i don't have a really strong opinion about that.